iogt/templates/core/tags/section_listing_homepage.html

```diff
@@ -1,19 +1,64 @@
-{% load wagtailcore_tags static i18n %}
+{% load wagtailcore_tags wagtailimages_tags comments static core_tags i18n %}
 
 {% if sections %}
-<div id="section_menu" class="section-menu">
-  <div class="section-heading">
-    <a href="#"><img src="{% static 'img/iogt_logo_2.png' %}" class="logo" alt="The Internet of Good Things" width="40%" /></a>
-  </div>
-  <ul class="menu-list">
-    {% for section in sections %}
-    <li class="menu-list__item menu-list__item{{section.get_effective_extra_style_hints}}">
-      <a href="{% pageurl section %}">
-        {{section.title}}
-        <span class="menu-list__desc">{{section.description}}</span>
-      </a>
-    </li>
-    {% endfor %}
-  </ul>
-</div>
+<h3 class="heading heading--homepage">{% trans "Recommended" %}</h3>
+<div class="section-list section-list__homepage">
+  {% for section in sections %}
+    {% load_descendant_articles_for_section section featured_in_homepage=True as featured_articles %}
+    {% if featured_articles %}
+    <div class="section-list__item section-list__item{{section.get_effective_extra_style_hints}}">
+      <ul class="article-list">
+      {% for article in featured_articles %}
+      <li class="article-list__item">
```

Comment:
Hmmm I think I don't want to duplicate the comment counter which doesn't really have an effect on whether the story is first.